micrometer-core/src/main/java/io/micrometer/core/annotation/Timed.java

```diff
@@ -22,6 +22,9 @@
 @Retention(RetentionPolicy.RUNTIME)
 @Inherited
 public @interface Timed {
+    /**
+     * The name of the metric 
```

Comment:
This is good. "name of the Timer" would be even more precise.

Reply:
What about "The name of the timer metric"?

Reply:
Sounds good to me